.github/workflows/ci.yml

```diff
@@ -32,8 +32,11 @@ jobs:
       run: printenv
       shell: bash
 
+    - name: 'Upgrade pip'
+      run: python -m pip install --upgrade pip
+
     - name: 'Install dependencies'
-      run: python -m pip install --upgrade -r requirements.txt
+      run: pip install -e .[full]
```

Comment:
@ihnorton any comments on this pattern? @Shelnutt2 is a little bit worried on this.

Reply:
Guys @ihnorton , @Shelnutt2 any news on the above?

Reply:
👍